app/src/main/java/org/mifos/mobile/ui/activities/HomeActivity.kt

```diff
@@ -117,16 +116,13 @@ class HomeActivity :
         lifecycleScope.launchWhenStarted {
             viewModel.userDetailUiState.collect {
                 when (it) {
-                    is UserDetailUiState.Loading -> showProgress()
-                    is UserDetailUiState.ShowUserDetails -> {
+                    is UserDetailState.Loading -> showProgress()
```

Comment:
should be UserDetailUiState

Reply:
As i have shifted the module to, new module, i changed the name to userdetailstate insted of userdetailuistate

Reply:
delete previous UserDetailUiState, 
And this UserDetailUiState should be in viewModel